js/main.js

```diff
@@ -0,0 +1,30 @@
+const lBars1 = document.querySelector('.line1__bars-menu');
+const lBars2 = document.querySelector('.line2__bars-menu');
+const lBars3 = document.querySelector('.line3__bars-menu');
+const listBars = document.querySelector('.list_bar-menu');
+
+document.querySelector('#bmenu').addEventListener('click', () => {
+  lBars1.classList.toggle('activeline1__bars-menu');
+  lBars2.classList.toggle('activeline2__bars-menu');
+  lBars3.classList.toggle('activeline3__bars-menu');
+  listBars.toggle('deploy');
```

Comment:
- [ ]  You javascript has some issue that you need to fix. When I click on your humburg menu, I  am getting an error in in console. and this is blocking your navigations for small device to appear. 

- [ ]  You should use same cross `x`similar to the one from figma design. 
![image](https://user-images.githubusercontent.com/43877689/174755821-4d68216f-dc22-43b1-84dc-ef96c600e1a1.png)